src/rgw/rgw_common.h

```diff
@@ -762,6 +763,7 @@ struct rgw_bucket {
     ::encode(index_pool, bl);
     ::encode(data_extra_pool, bl);
     ::encode(tenant, bl);
+    ::encode(name_space, bl);
```

Comment:
if 9 was merged upstream then we need to bump this one to 10, even if 9 didn't add any encoding changes